docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding.md

```diff
@@ -0,0 +1,149 @@
+
+There are two primary roles: *operator* and *merchant*:
+* An operator is a company that owns and administers the platform.
+* A merchant is a business entity or individual that sells products on the operator's platform. The operator is responsible and engaged with the merchants to determine how they conduct their operations on the platform. The operator can serve as a merchant as well.
+
+The onboarding process consists of seven steps. Each step requires specific actions that merchants, operators, or both need to take. 
+
+If you run into any issues, please get in touch with your Spryker contact.
+
+## Marchant onboarding process
+
+The merchant onboarding process consists of the following steps:
+
+1. The merchant puts in a request to join the marketplace.
+2. The operator verifies the merchant's company and shares a contract.
+3. The operator approves the merchant and provides them with access to the Merchant Portal to start completing their public profile.
+4. The merchant creates products and offers in the Merchant Portal. Alternatively, the operator can help the merchant set up the process for automatically importing products and offers from a CSV template.
+5. The operator decides what kind of payment process to implement—for example, whether or not to use a *Payment Service Provider (PSP)*.
+6. An order management process that includes the process for fulfillment, including delivery, shipping, and returns, is established for merchants through Spryker State Machine.
+7. A final check on the public profile, products, and offer quality is conducted. The operator activates merchants and their products and offers in Back Office. Both can be activated via a data importer as well.
+
+Each step is described in the following sections.
+
+### 1) Merchant: Put in a request to join
+
+The official onboarding starts with a merchant requesting to join the marketplace. 
+
+To request a new merchant account, use one of the following options:
+* Build a landing page where the potential merchants can find the email address or a form to contact the operator to join the marketplace. In some cases, the operator may contact their partners and inform them about the marketplace.
+* The merchant can use the page to self-register. Once complete, the data is sent to the operator for review (the feature is on the product roadmap and will be available soon).
+* The preceding two options can be combined.
+
+### 2) Operator: Verify (the KYC process) and register a merchant
+
+1. Gather data a merchant provides, such as a registration number and the company behind the merchant, including representatives' information and their IDs.
+2. To identify the trust level and the need for additional information, validate your merchant through public registers. 
+
+{% info_block infoBox %}
+
+To accelerate the validation process, you can use a relevant Spryker technology partner—for example, detected.app.
```

Comment:
@mikost7 what's "detected.app"? Can you provide a link to it? I couldn't find anything about it on our documentation portal.

Reply:
```suggestion
To accelerate the validation process, you can use relevant Spryker technology partners. For more details, reach out to your Spryker contact.
```